core/app/models/spree/order.rb

```diff
@@ -787,7 +787,7 @@ def after_cancel
     end
 
     def send_cancel_email
-      OrderMailer.cancel_email(self.id).deliver_later
+      OrderMailer.cancel_email(self).deliver_later
```

Comment:
I think we should use the `#id` here so that we don't send an AR object through AJ.